cluster/gce/config-test.sh

```diff
@@ -53,6 +53,8 @@ NODE_LOCAL_SSDS_EXT=${NODE_LOCAL_SSDS_EXT:-}
 NODE_ACCELERATORS=${NODE_ACCELERATORS:-""}
 REGISTER_MASTER_KUBELET=${REGISTER_MASTER:-true}
 KUBE_APISERVER_REQUEST_TIMEOUT=300
+# Increase initial delay for the apiserver liveness probe, to avoid prematurely tearing it down
+KUBE_APISERVER_LIVENESS_PROBE_INITIAL_DELAY_SEC=${KUBE_APISERVER_LIVENESS_PROBE_INITIAL_DELAY_SEC:-45}
```

Comment:
As long as ETCD liveness is part of the KAS healthz checks I would keep them in sync on the initial delay.

Reply:
Sure, that makes sense. I've made the corresponding bump for etcd.

Reply:
I verified the etcd configs on GCE deployed e2e cluster via manual inspection of the manifest files. Resolving this.